static/js/whyis.js

```diff
@@ -441,41 +441,63 @@ function whyis() {
         SmartFacetConstructor.prototype.setSelectedValue = setSelectedValue;
         SmartFacetConstructor.prototype.getConstraint = getConstraint;
 
+        SmartFacetConstructor.prototype.getState = function() {
```

Comment:
I'm pretty sure you could use an ES6 class here. Worth trying.

Reply:
This is building off of an existing framework that uses this approach. I need to remain compatible with it. We are partway through refactoring it away, so after that, we can take a look.